packages/shared/forks/object-assign.inline-umd.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ */
+
+const hasOwnProperty = Object.prototype.hasOwnProperty;
+
+const _assign = function(to, from) {
+  for (let key in from) {
+    if (hasOwnProperty.call(from, key)) {
+      to[key] = from[key];
+    }
+  }
+};
+
+export default Object.assign ||
+  function(target, sources) {
+    if (target == null) {
+      throw new TypeError('Object.assign target cannot be null or undefined');
```

Comment:
Do we actually care about this? Would we expect to ever happen with our usage?

Reply:
Maybe if it's a bug. Meh. I mostly just took it as-is from FB.